docker-compose.test.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.8'
+
+services:
+  tax-report-test-db:
+    image: postgres:15.1-alpine
+    environment:
+      POSTGRES_DB: tax-report
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: 123
+    ports:
+      - 5433:5432
+    volumes:
+      - tax-report-test-db-data:/var/lib/postgresql/data
+  adminer:
+    image: adminer
+    restart: always
+    ports:
+      - 8080:8080
+volumes:
+  tax-report-test-db-data:
```

Comment:
not sure if it's a good idea to name it tax-report-test-db-data instead of just db-data